features/step_definitions/catalogue_steps.rb

```diff
@@ -41,3 +41,9 @@
 Then(/^I see #{MAYBE_VAR} in the search summary text$/) do |value|
   find(:xpath, "//*[@class='search-summary']/em[1]").text().should == normalize_whitespace(value)
 end
+
+When /^I tick a random filter$/ do
+  checkbox = all(:xpath, "//form[@action='/g-cloud/search']//input[@type='checkbox']").sample
+  page.check(checkbox[:id])
+  puts "Filter ticked: #{checkbox[:name]}"
```

Comment:
"Filter value"?  
[Would make it consistent](https://github.com/alphagov/digitalmarketplace-functional-tests/blob/b2c109e7d4f9a6957f59fc8c0a2c4ae3ee608af5/features/support/helpers.rb#L61) with the other print statements.